CHANGELOG.md

```diff
@@ -62,6 +62,8 @@ Main (unreleased)
 - Use Go 1.20 for builds. Official release binaries are still produced using Go
   1.19. (@rfratto)
 
+- Use river squash for otelcol.processor.tail_sampling code which doesn't impact the config or docs (@erikbaranowski)
```

Comment:
It's not a user facing change, so we wouldn't document it in the changelog

Reply:
I was wondering that but took a stab. Should be fixed now.